tests/test_cmdline.py

```diff
@@ -38,6 +38,14 @@ def _check_no_warnings(module):
     _check_no_warnings("distutils")
 
 
+def test_commandline_ospathsep(tmpdir):
+    path = str(tmpdir.join("pathsepvenv" + os.pathsep + "0"))
+    assert not os.path.exists(path)
+    ret = subprocess.call([sys.executable, VIRTUALENV_SCRIPT, path])
```

Comment:
pass in the tmpdir as cwd

Reply:
`test_commandline_basic` takes the `tmpdir`  and joins it with `venv` so I followed that same pattern, but have changed the name.